rpm/prebuild.sh

```diff
@@ -2,4 +2,17 @@
 
 set -exu  # Strict shell (w/o -o pipefail)
 
-curl -LsSf https://www.tarantool.io/release/1.10/installer.sh | sudo bash
+os=""
+dist=""
+if [ -f /etc/os-release ]; then
+    . /etc/os-release
+    os="$ID"
+    dist="$VERSION_ID"
+fi
+
+if [[ "$os" == "almalinux" && "$dist" == 9* ]]; then
+    curl -LsSf https://tarantool.io/release/3/installer.sh | \
+    sed '2s/^/echo 8 > \/etc\/centos-release\n/' | sudo bash
+else
+    curl -LsSf https://tarantool.io/release/1.10/installer.sh | sudo bash
```

Comment:
Please, add a comment why we install 1.10 instead of 3.
```suggestion
    curl -LsSf https://tarantool.io/release/3/installer.sh | sudo bash
```

Reply:
Added.